Console/Command/UpgradeVersionShell.php

```diff
@@ -28,8 +28,23 @@ public function main() {
 
     public function _ug110()
     {
-        $query = "alter table cm_rcauth_sources alter column client_secret type varchar(1024) using client_secret::varchar(1024)";
-        $this->RcauthSource->query($query);
-        $this->out('Change applied: ' . $query);
+        $err = false;
+        $query = "alter table cm_rcauth_sources alter column client_secret type varchar(1024) using client_secret::varchar(1024);";
+        $query .= "alter table cm_rcauth_sources add scopes varchar(256)";
+        $dbc = $this->RcauthSource->getDataSource();
+        $dbc->begin();
+        try {
+            $this->RcauthSource->query($query);
+        }
+        catch(Exception $e) {
+            $dbc->rollback();
+            $err = true;
+            $this->out($e->getMessage());
+        }
+        if(!$err) {
```

Comment:
This is not needed. Add the commit in the try block. As soon as an error is thrown then the catch block will take control